manager/handlers/application.go

```diff
@@ -159,10 +179,23 @@ func (h *Handlers) GetApplications(ctx *gin.Context) {
 		return
 	}
 
+	// Validate pagination parameters
+	if query.Page < 0 {
```

Comment:
The parameter verification has been completed in the annotation, refer to https://github.com/dragonflyoss/dragonfly/blob/main/manager/types/audit.go#L40.